com.sayonara_player.Sayonara.json

```
@@ -0,0 +1,141 @@
+{
+    "app-id": "com.sayonara_player.Sayonara",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "sayonara",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.mpris.MediaPlayer2.Player",
+        "--own-name=org.mpris.MediaPlayer2.sayonara",
+        "--own-name=org.kde.*",
+        "--device=all",
+        "--filesystem=xdg-run/gvfs"
+    ],
+    "rename-appdata-file": "sayonara.appdata.xml",
+    "rename-desktop-file": "com.sayonara-player.Sayonara.desktop",
+    "rename-icon": "sayonara",
+    "modules": [
+        {
+            "config-opts": [],
+            "buildsystem": "autotools",
+            "name": "soundtouch",
+            "sources": [
+                {
+                    "commit": "762f56024b7ade81f6565903161dffec0ad4741e",
+                    "tag": "2.2",
+                    "type": "git",
+                    "url": "https://gitlab.com/soundtouch/soundtouch.git"
+                }
+            ]
+        },
+        {
+            "config-opts": [],
+            "buildsystem": "meson",
+            "name": "gstreamer",
+            "sources": [
+                {
+                    "commit": "a42fe476d3ee5576921f67a331464065ec33b9a4",
+                    "tag": "1.18.3",
+                    "type": "git",
+                    "url": "https://gitlab.freedesktop.org/gstreamer/gstreamer.git"
+                }
+            ]
+        },
+        {
+            "config-opts": [],
```

```suggestion
            "config-opts": [],
```
you probably want to at least drop building the docs/tests/examples here and for the other gst stuff

---

Fixed with e74f3b4